src/templates/items/weapon.hbs

```diff
@@ -18,75 +18,106 @@
 
         {{!-- Details Tab --}}
         <div class="tab details" data-group="primary" data-tab="details">
-            <h3 class="form-header noborder">{{ localize "SFRPG.Items.Weapon.Details" }}</h3>
 
-            {{!-- Weapon Type --}}
-            <div class="form-group">
-                <label>{{ localize "SFRPG.Items.Weapon.Type" }}</label>
-                <select name="system.weaponType">
-                    {{#select itemData.weaponType}}
-                    {{#each config.weaponTypes as |name type|}}
-                    <option value="{{type}}">{{name}}</option>
-                    {{/each}}
-                    {{/select}}
-                </select>
-            </div>
+            {{!-- Details Tab Navigation --}}
+            <nav class="sheet-subnavigation subtabs" data-group="subdetails">
+                <a class="item active" data-tab="properties">Properties</a>
```

Comment:
Need to figure out how to get this to highlight by default correctly, while switching font weight and bottom border color properly

Reply:
Fixed by removing the highlight since it doesn't seem necessary to me